### PR TITLE
controlnet updates

### PR DESCRIPTION
**Pull request by GitWilburn**
--Add FaceID to controlnet list
--Added support for t2i versions of some of the models (faster)
--Added better resolution support which should help with XL models
--Moved cropping settings to the image section instead of the pre-processor section
--Added Sketch

### CODE REVIEW

File: library/built-in/SDUI.ts [hunk 1]
```diff
@@ -160,12 +161,10 @@ app({
             if (ui.highResFix.saveIntermediaryImage) {
                 graph.SaveImage({ images: graph.VAEDecode({ samples: latent, vae }) })
             }
-            latent = graph.LatentUpscale({
-                samples: latent,
-                crop: 'disabled',
-                upscale_method: 'nearest-exact',
-                height: ui.latent.size.height * ui.highResFix.scaleFactor,
-                width: ui.latent.size.width * ui.highResFix.scaleFactor,
+            latent = graph.NNLatentUpscale({
```
Comment (GitWilburn) [inline library/built-in/SDUI.ts:164]: @rvion I accidentally included this in the commit. Bit it's not a bad change. I discovered the NNLatentUpscale and really like it. But feel free to exclude it if you want.

Comment (rvion) [reply library/built-in/SDUI.ts:164]: Oh, didn't know about it; Does it require a custom node?

Comment (GitWilburn) [reply library/built-in/SDUI.ts:164]: Shoot, yeah. So maybe not good for default latent upscale.

https://github.com/Ttl/ComfyUi_NNLatentUpscale

**Comment by rvion** [comment]
great ! I'll merge as-is cause I added a bunch of things on top

**Comment by rvion** [comment]
image
lol, can't even merge myself since this is marked as draft; I'll merge from the CLI with my commit on top. closing this PR here, since it's "manually merged"

**Pull request closed without merging**